mirdata/core.py

```diff
@@ -283,6 +283,58 @@ def choice_multitrack(self):
         """
         return self.multitrack(random.choice(self.mtrack_ids))
 
+    def get_tracks_splits(self, splits, seed=42):
+        """Split the tracks dataset for training, validation, test, etc
+
+        Args:
+            splits (list of float): a list of floats that should sum up 1. It will return as many splits as elements in the list
+            seed (int): the seed used for the random generator, in order to enhance reproducibility. Defaults to 42
+
+        Returns:
+            list of lists: a list of lists containing the elements in each split
+        """
+
+        if not np.isclose(np.sum(splits), 1):
+            raise ValueError(
+                "Splits values should sum up to 1. Given {} sums {}".format(
+                    splits, np.sum(splits)
+                )
+            )
+
+        rng = np.random.default_rng(seed=seed)  # The random generator
+        random_track_ids = rng.permutation(self.track_ids)
```

Comment:
Maybe you can add a condition to check that the dataset has tracks? and the same for multitracks? to avoid triggering an error hard to debug in this line or on the code below?

Reply:
Done! Unfortunately, my computer is kinda Frankenstein and I can not run the tests on my local setup. I will try to find some alternative where I can run the write & run tests and meet the coverage expectations.